app/models/reservation.rb

```diff
@@ -0,0 +1,10 @@
+class Reservation < ApplicationRecord
+  belongs_to :user
+  belongs_to :glamping
+
+  validates :glamping_id, presence: true
+  validates :user_id, presence: true
+  validates :reservation_date, presence: true
+  validates :due_date, presence: true
```

Comment:
[OPTIONAL] IS there any other validation yo ucan think of? should due date be only in a certain range? Like not before the day of reservation?